diff --git a/libvita2d/Makefile b/libvita2d/Makefile
index 3fc3633..86a403d 100644
--- a/libvita2d/Makefile
+++ b/libvita2d/Makefile
@@ -9,10 +9,10 @@ SHADERS    = shader/compiled/clear_v_gxp.o shader/compiled/clear_f_gxp.o \
              shader/compiled/texture_v_gxp.o shader/compiled/texture_f_gxp.o \
              shader/compiled/texture_tint_f_gxp.o
 
-PREFIX  ?= ${VITASDK}/arm-vita-eabi
-CC      = arm-vita-eabi-gcc
-AR      = arm-vita-eabi-ar
-CFLAGS  = -Wl,-q -Wall -O3 -I$(INCLUDES) -I$(VITASDK)/arm-vita-eabi/include/freetype2 -ffat-lto-objects -flto
+PREFIX  ?= ${DOLCESDK}/arm-dolce-eabi
+CC      = arm-dolce-eabi-gcc
+AR      = arm-dolce-eabi-ar
+CFLAGS  = -Wl,-q -Wall -O3 -I$(INCLUDES) -I$(DOLCESDK)/arm-dolce-eabi/include/freetype2
 ASFLAGS = $(CFLAGS)
 
 all: $(TARGET_LIB)
