diff --git a/Makefile.vita b/Makefile.vita
index 81b5a2b..2230b0b 100644
--- a/Makefile.vita
+++ b/Makefile.vita
@@ -5,10 +5,10 @@ COMMON_OBJS = SDLnet.o \
 	SDLnetUDP.o \
 	SDLnetvita.o
 
-PREFIX  = arm-vita-eabi
+PREFIX  = arm-dolce-eabi
 CC      = $(PREFIX)-gcc
 AR      = $(PREFIX)-ar
-CFLAGS  = -Wl,-q -Wall -O3 -Iinclude -I$(VITASDK)/arm-vita-eabi/include/SDL
+CFLAGS  = -Wl,-q -Wall -O3 -Iinclude -I"$(DOLCESDK)/arm-dolce-eabi/include/SDL"
 ASFLAGS = $(CFLAGS)
 
 LIBS    += -lSDL
@@ -20,8 +20,9 @@ clean:
 	@rm -rf $(TARGET_LIB) $(COMMON_OBJS)
 
 install: $(TARGET_LIB)
-	@cp  $(TARGET_LIB) $(VITASDK)/arm-vita-eabi/lib
-	@mkdir -p "$(VITASDK)/arm-vita-eabi/include/SDL"
-	@cp SDL_net.h "$(VITASDK)/arm-vita-eabi/include/SDL"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/arm-dolce-eabi/lib"
+	@cp  $(TARGET_LIB) "$(DESTDIR)$(DOLCESDK)/arm-dolce-eabi/lib"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/arm-dolce-eabi/include/SDL"
+	@cp SDL_net.h "$(DESTDIR)$(DOLCESDK)/arm-dolce-eabi/include/SDL"
 	@echo "Installed!"
 
diff --git a/SDLnetvita.c b/SDLnetvita.c
index 531683f..a972d4c 100644
--- a/SDLnetvita.c
+++ b/SDLnetvita.c
@@ -30,6 +30,8 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <netdb.h>
+#include <psp2/sysmodule.h>
+#include <psp2/libnetctl.h>
 
 #include "SDL_net.h"
 #include "SDLnetvita.h"
diff --git a/SDLnetvita.h b/SDLnetvita.h
index 1e6ba7b..0972a53 100644
--- a/SDLnetvita.h
+++ b/SDLnetvita.h
@@ -24,7 +24,7 @@
 #pragma once
 
 #include <sys/select.h>
-#include <vitasdk.h>
+#include <psp2/net.h>
 
 // These are all defined in SDLnetvita.c
 
