diff --git a/Makefile.vita b/Makefile.vita
index 64a1af1..eeaee81 100644
--- a/Makefile.vita
+++ b/Makefile.vita
@@ -24,10 +24,10 @@ COMMON_OBJS = dynamic_flac.o \
 	playwave.o \
 	wavestream.o
 
-PREFIX  = arm-vita-eabi
+PREFIX  = arm-dolce-eabi
 CC      = $(PREFIX)-gcc
 AR      = $(PREFIX)-ar
-CFLAGS  = -Wl,-q -Wall -O3 -I$(VITASDK)/arm-vita-eabi/include/SDL -DOGG_MUSIC -DWAV_MUSIC -DMOD_MUSIC
+CFLAGS  = -Wall -O3 -I$(DOLCESDK)/$(PREFIX)/include/SDL -DOGG_MUSIC -DWAV_MUSIC -DMOD_MUSIC
 
 ASFLAGS = $(CFLAGS)
 
@@ -40,8 +40,9 @@ clean:
 	@rm -rf $(TARGET_LIB) $(COMMON_OBJS)
 
 install: $(TARGET_LIB)
-	@cp  $(TARGET_LIB) $(VITASDK)/arm-vita-eabi/lib
-	@mkdir -p "$(VITASDK)/arm-vita-eabi/include/SDL"
-	@cp SDL_mixer.h "$(VITASDK)/arm-vita-eabi/include/SDL"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/lib"
+	@cp  $(TARGET_LIB) "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/lib"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/include/SDL"
+	@cp SDL_mixer.h "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/include/SDL"
 	@echo "Installed!"
 
