diff --git a/Makefile.vita b/Makefile.vita
index fc265e0..94c593c 100644
--- a/Makefile.vita
+++ b/Makefile.vita
@@ -17,11 +17,11 @@ COMMON_OBJS = IMG.o \
 	IMG_xxx.o \
 	showimage.o
 
-PREFIX  = arm-vita-eabi
+PREFIX  = arm-dolce-eabi
 CC      = $(PREFIX)-gcc
 AR      = $(PREFIX)-ar
 CFLAGS  = -Wl,-q -Wall -O3 -Iinclude \
-				-I$(VITASDK)/arm-vita-eabi/include/SDL2 \
+				-I$(DOLCESDK)/$(PREFIX)/include/SDL2 \
 				-DLOAD_JPG \
 				-DLOAD_PNG \
 				-DLOAD_BMP \
@@ -46,7 +46,8 @@ clean:
 	@rm -rf $(TARGET_LIB) $(COMMON_OBJS)
 
 install: $(TARGET_LIB)
-	@cp  $(TARGET_LIB) $(VITASDK)/arm-vita-eabi/lib
-	@mkdir -p "$(VITASDK)/arm-vita-eabi/include/SDL2"
-	@cp SDL_image.h "$(VITASDK)/arm-vita-eabi/include/SDL2"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/lib"
+	@cp  $(TARGET_LIB) "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/lib"
+	@mkdir -p "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/include/SDL2"
+	@cp SDL_image.h "$(DESTDIR)$(DOLCESDK)/$(PREFIX)/include/SDL2"
 	@echo "Installed!"
